api/demos/All-AoU-Participants.json

```diff
@@ -0,0 +1,276 @@
+{
+  "type": "cohort",
+  "cohort": {
+    "name": "All AoU Participants",
+    "description": "demo",
+    "type": "demo",
+    "criteria": {
+      "includes": [
+        {
+          "id":"includes_osphk7pib",
+          "items":[
+            {
+              "id":"items_jz17loxbt",
+              "type":"PPI",
+              "searchParameters": [
+                {
+                  "parameterId":"param325062",
+                  "name":"Overall Health",
+                  "type":"PPI",
+                  "subtype":"OVERALLHEALTH",
+                  "group":true
+                },{
+                  "parameterId":"param324923",
+                  "name":"Lifestyle",
+                  "type":"PPI",
+                  "subtype":"LIFESTYLE",
+                  "group":true
+                },{
+                  "parameterId":"param324836",
+                  "name":"The Basics",
+                  "type":"PPI",
+                  "subtype":"BASICS",
+                  "group":true
+                }
+              ],
+              "modifiers":[]
+            },{
+              "id":"items_msuckl2qf",
+              "type":"DEMO",
+              "searchParameters":[
+                {
+                  "parameterId":"param8532",
+                  "name":"Female",
+                  "type":"DEMO",
+                  "subtype":"GEN",
+                  "group":false,
+                  "conceptId":8532
+                },{
+                  "parameterId":"param8507",
+                  "name":"Male",
+                  "type":"DEMO",
+                  "subtype":"GEN",
+                  "group":false,
+                  "conceptId":8507
```

Comment:
I think at this point just using 'Male' and 'Female' will be most accurate per our slack conversation. So I would remove the rest of this cohort definition.

```
{
  "type": "cohort",
  "cohort": {
    "name": "All AoU Participants",
    "description": "demo",
    "type": "demo",
    "criteria": {
      "includes": [
        {
          "id":"includes_osphk7pib",
          "items":[
            {
              "id":"items_msuckl2qf",
              "type":"DEMO",
              "searchParameters":[
                {
                  "parameterId":"param8532",
                  "name":"Female",
                  "type":"DEMO",
                  "subtype":"GEN",
                  "group":false,
                  "conceptId":8532
                },{
                  "parameterId":"param8507",
                  "name":"Male",
                  "type":"DEMO",
                  "subtype":"GEN",
                  "group":false,
                  "conceptId":8507
                }
              ],
              "modifiers":[]
            }
          ],
          "temporal":false
        }
      ],
      "excludes":[]
    }
  }
}
```